### ISSUE TITLE
[amenity-symbols] mapnik: render highway=ford on ways

### ISSUE THREAD

**Issue by tomhughes** [tomhughes/trac-tickets#3007]
**[Submitted to the original trac issue database at 8.17am, Tuesday, 25th May 2010]**

Hi,

highway=ford already gets rendered for nodes, but not for ways although about one third of the tag is applied to ways.

Regards, Alex

**Comment by tomhughes** [comment]
**Author: Ldp**
**[Added to the original trac issue at 11.37pm, Thursday, 3rd June 2010]**

I already have this on the Haiti map, but when I went to apply that to the generic map, I found it produced unexpected results. Actually, on the Haiti map, I render them about the same as unclassified, but with a bluish hue and broken casing[1]. In Haiti, that works, but elsewhere in the world, these highway=ford connect to highway=track, and the result is a displeasing mix of a generic track and a bluish unclassified where the ford is.

This, of course, is the unfortunate effect of the upper class highway=ford, which carries no other information about what type of road it actually is, versus the more logical lower class ford=yes[2] on a highway=track (or unclassified, or whichever other type). Unfortunately, highway=ford is still promoted on Map Features, while the alternative ford=* isn't.

[1] http://haiti.openstreetmap.nl/?zoom=16&lat=18.21342&lon=-73.87641&layers=B00FFFFFFFFFF

[2] http://wiki.openstreetmap.org/wiki/Key:ford
